suite-native/graph/src/components/TransactionEventTooltip.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import Animated, { FadeIn, FadeOut } from 'react-native-reanimated';
+import { Dimensions } from 'react-native';
+
+import { G, N } from '@mobily/ts-belt';
+
+import { Card, Text } from '@suite-native/atoms';
+import { CryptoAmountFormatter, SignValueFormatter } from '@suite-native/formatters';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { GroupedBalanceMovement } from '@suite-common/graph';
+
+export type TransactionEventTooltipProps = GroupedBalanceMovement['payload'] & {
+    eventX: number | null;
+    eventY: number | null;
+};
+
+const ANIMATION_DURATION = 200;
+const SCREEN_WIDTH = Dimensions.get('screen').width;
+
+const TooltipContainerStyle = prepareNativeStyle<{ x: number; y: number }>((_, { x, y }) => ({
+    position: 'absolute',
+    left: x + 25,
+    top: N.clamp(0, 100, y), // The clamping prevents an Y axis overflow.
+    zIndex: 10000,
```

Comment:
nitpick: Do we really need to use this much? 😅

Reply:
Actually, it is not needed at all :D. It is the remainder from the old original solution. Removed this style prop in 90955f4fc.